interfaces/automated-testing/scd/design/u-space/README.md

```diff
@@ -0,0 +1,12 @@
+The tests in this group validate requirements unique to U-space flight
+authorisations.
```

Comment:
May I suggest to link to the [Article 10 of the regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32021R0664&from=EN#d1e1041-161-1)?

Reply:
Done.